### PR TITLE
fixed navigation scrollbar height

### PR DESCRIPTION
**Pull request by DwinaTech**

### CODE REVIEW

**Comment by kkarimi** [comment]
@DwinaTech Please include a screenshot to see the difference here

File: frontend/src/helpers.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ function reformatCategoryName(name) {
 
 function categoryNameMaker(name) {
   return name
-    ? name.replace('/', '').replace("services/", '').replace("/add", '').replace(/\b\w/g, l => l.toUpperCase())
+    ? name.replace(/[/]/g, ' ').replace("services/", '').replace("/add", '').replace(/\b\w/g, l => l.toUpperCase())
```
Comment (kkarimi) [inline frontend/src/helpers.js]: Why did you go with this approach instead of using the HTML char that we discussed? does this work better?

Comment (DwinaTech) [reply frontend/src/helpers.js]: yeah this one work perfectly

File: frontend/src/helpers.js [hunk 2]
```diff
@@ -144,8 +144,9 @@ function reformatCategoryName(name) {
 }
 
 function categoryNameMaker(name) {
+  // This [/] will replace all of / to empty space
```
Comment (kkarimi) [inline frontend/src/helpers.js]: All the rules, not just one....

Comment (DwinaTech) [reply frontend/src/helpers.js]: Done

**Pull request merged**